params/cube_corners.json

```diff
@@ -0,0 +1,64 @@
+{
+    "id": "cube_corners",
+    "name": "Cube Corner Sphere Inversions",
+    "description": "Similar to tangent_spheres, but this time it's 8 spheres at the corners of the unit cube",
+    "algorithm": "chaos_sets",
+    "plotter": {
+        "radius": 4
+    },
+    "cluster": {
```

Comment:
I wasn't quite happy with the look of the result here. revisit this now that I have the many clusters